wolfcrypt/src/fe_448.c

```diff
@@ -1836,6 +1836,8 @@ static WC_INLINE void fe448_mul_8(sword32* r, const sword32* a, const sword32* b
     sword64 t13  = (sword64)a[ 6] * b[ 7];
     sword64 t113 = (sword64)a[ 7] * b[ 6];
     sword64 t14  = (sword64)a[ 7] * b[ 7];
+    sword64 o;
```

Comment:
@SparkiDev , when you are back can you review the scripts repo to see if it needs this fix also?